welcome.md

```diff
@@ -2,30 +2,33 @@
 icon: home
 order: 10000
 title: Introduction
+# layout: central
 ---
 
-### Welcome to Collar!
+# Introducing Votre
 
-Collar is the best way to borrow against crypto.
+### A Smarter Way to Borrow Against Crypto
 
-Collar is completely non-custodial and does not rely on liquidations to remain solvent. Collar offers the highest LTVs in DeFi without introducing credit or counterparty risk. Collar is powered by solvers instead of liquidators as well as other DeFi primitives like Uniswap v3.
+Votre is a liquidation-free, non-custodial lending protocol that allows users to borrow against their crypto assets with high loan-to-value (LTV) ratios, no interest, and potential tax advantages. Unlike traditional lending platforms, Votre removes directional risk upfront, ensuring borrowers never face forced liquidation.
 
-Collar is short for “COLlaborative Lending ARrangement” (Collar). It also happens to match the name of an options strategy with a similar payout profile.
+By leveraging solvers instead of liquidators, Votre guarantees performance up to a return cap, allowing borrowers to access liquidity while maintaining exposure to their assets.
 
-Collars are agreed upon offchain and settled onchain. For those of you Collar OG's, the new architecture is intent-based, relying on offchain, whitelisted solvers to ensure users receive prompt, competitive pricing on proposed Collars. This is called the Offchain Intent Platform throughout these docs.
+### Why Choose Votre?
 
-This documentation explains how Collar works and answers questions prospective borrowers may have. Borrowers can access Collar Protocol via the smart contracts directly or via the Collar Frontend, hosted at https://app.collarprotocol.xyz, launching later this year.
+- [x] **Liquidation-Free** – No risk of forced liquidation, as assets are swapped upfront.
+- [x] **Tax-Advantaged** – Borrowing can defer tax obligations in many jurisdictions.
+- [x] **Non-Custodial** – Users retain full control; positions can mature automatically.
+- [x] **Liquidity-Independent** – No reliance on unsustainable yield farming.
```

Comment:
provider liquidity is important for offering loans and rolls, and swap liquidity for opening and closing